app/services/form1010cg/auditor.rb

```diff
@@ -25,6 +25,10 @@ def self.metrics
       )
     end
 
+    def initialize(logger)
```

Comment:
```suggestion
    def initialize(logger = Rails.logger)
```
Just a suggestion, but this way the auditor can accept other loggers, but in the common uses it doesn't need too explicitly add it.